src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  div.className = 'notification';
+  div.classList.add(type);
+  div.style.top = posTop + 'px';
+  div.style.right = posRight + 'px';
+  div.visibility = 'visible';
```

Comment:
The line `div.visibility = 'visible';` is incorrect. It should be `div.style.visibility = 'visible';` to properly set the visibility of the div.